packages/data-explorer-ui/src/components/common/Banner/banner.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Typography } from "@mui/material";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { CallToActionButton } from "../Button/components/CallToActionButton/callToActionButton";
+import { Banner } from "./banner";
+import { BannerPrimary } from "./banner.styles";
+
+export default {
+  argTypes: {
+    children: { control: { disabled: true } },
+  },
+  component: Banner,
+  title: "Components/Common/Banner",
+} as ComponentMeta<typeof Banner>;
+
+const BannerPrimaryTemplate: ComponentStory<typeof Banner> = (args) => (
+  <Box sx={{ minWidth: 300 }}>
+    <BannerPrimary {...args} />
+  </Box>
+);
+
+export const BannerPrimaryStory = BannerPrimaryTemplate.bind({});
+BannerPrimaryStory.args = {
+  children: (
+    <>
+      <Typography component="span" variant="text-body-small-500">
+        Optional announcements text goes here followed by a
+      </Typography>{" "}
+      <CallToActionButton
```

Comment:
Let's use the CallToActionButton story here instead, and for the prop `ButtonElType` use `TextButtonWhite`.

Reply:
@imadx if we import the newly created TextButtonWhiteStory and replace ButtonElType={"TextButtonWhite"} with ButtonElType={TextButtonWhite}, I think this means we do not need to update or make changes to `CallToActionButton` component.

Reply:
Let's also use the `CallToActionButtonStory` instead of `CallToActionButton` component on line `30`.